notNeededPackages.json

```diff
@@ -1080,6 +1080,10 @@
             "libraryName": "consul",
             "asOfVersion": "2.0.0"
         },
+        "content-range": {
+            "libraryName": "content-range",
+            "asOfVersion": "2.0.0"
```

Comment:
I believe this should be `2.0.2`. Versions 2.0.0 and 2.0.1 have a `package.json` whose types field says `index.d.js`, but that file doesn't exist. 2.0.2 seems to be the first patch that fixes that.

Reply:
Done.